### PR TITLE
boards: shields: add MikroElektronika ACCEL 13 Click shield

### PR DESCRIPTION
**Pull request by k0d**
Add shield definition for the MikroElektronika ACCEL 13 Click. The ACCEL 13 Click carries a IIS2DLPC ultra-low power triaxial accelerometer sensor in a mikroBUS form factor.

### CODE REVIEW

**Comment by carlescufi** [comment]
Thanks for the PR! Please take a look at [our commit message guidelines](https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-message-guidelines) to find out how to format your commit messages, and at [our contribution workflow](https://docs.zephyrproject.org/latest/contribute/guidelines.html#contribution-workflow) to understand how to update your Pull Request.

**Review by erwango — DISMISSED** [review]

**Review by jfischer-no — DISMISSED** [review]

**Comment by kartben** [comment]
@erwango @jfischer-no I was about to make a similar comment on #62708 but don't we want shields to define aliases (here, `accel0`) and chosens (there, `zephyr, display`). This at least seems to be a widely adopted convention for most/all other shields, so maybe even worth documenting too?

**Comment by erwango** [comment]
> @erwango @jfischer-no I was about to make a similar comment on #62708 but don't we want shields to define aliases (here, `accel0`) and chosens (there, `zephyr, display`). This at least seems to be a widely adopted convention for most/all other shields, so maybe even worth documenting too?

That indeed looks a good thing. I just wonder a bout conflict handling with board's own aliases and chosen then. It could be ok to overwrite but at least we need to agree formally about it. Would you mind raising an issue for this point (so we can discuss and finalize the decision there) ?

**Comment by jfischer-no** [comment]
> @erwango @jfischer-no I was about to make a similar comment on #62708 but don't we want shields to define aliases (here, `accel0`) and chosens (there, `zephyr, display`). This at least seems to be a widely adopted convention for most/all other shields, so maybe even worth documenting too?



> > @erwango @jfischer-no I was about to make a similar comment on #62708 but don't we want shields to define aliases (here, `accel0`) and chosens (there, `zephyr, display`). This at least seems to be a widely adopted convention for most/all other shields, so maybe even worth documenting too?
> 
> That indeed looks a good thing. I just wonder a bout conflict handling with board's own aliases and chosen then. It could be ok to overwrite but at least we need to agree formally about it. Would you mind raising an issue for this point (so we can discuss and finalize the decision there) ?

This is already a practice, perhaps just add a note to the shield documentation that the user should be aware that alias/selected properties can be overwritten by a shield. FYI @mbolivar-ampere

File: boards/shields/mikroe_accel13_click/mikroe_accel13_click.overlay [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2023 Mark Olsson <mark@markolsson.se>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```
Comment (kartben) [inline boards/shields/mikroe_accel13_click/mikroe_accel13_click.overlay:6]: please add an alias for iis2dlpc_mikroe_accel13_click --> accel0 (see other shields)

**Review by kartben — CHANGES_REQUESTED** [review]

File: boards/shields/mikroe_accel13_click/mikroe_accel13_click.overlay (hunk 1 above)
Comment (k0d) [reply boards/shields/mikroe_accel13_click/mikroe_accel13_click.overlay:6]: I've done this now...I didn't add anything to the docs though, I wasn't sure what to write and where.

**Review by jfischer-no — APPROVED** [review]

**Review by kartben — APPROVED** [review]

**Comment by k0d** [comment]
Will this be merged soon? I'm just wondering what the procedure is and who decides when things should get merged.

**Pull request merged**